README.md

```diff
@@ -3,7 +3,7 @@
 <div align="center">
   <br/>
 
-  <h3><b> My Todo list</b></h3>
+  <h3><b>Todo list</b></h3>
 
 </div>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228906292-23177c1a-5d7b-44bc-aa4a-4fdb99c055c9.png)

- [ ] It would be nice if you make your title more descriptive to make your pull request more professional 👍 You can use **To Do list: add & remove** instead of **modify readme file** which is more descriptive and tells what is happening in this pull request.